README.md

```diff
@@ -156,19 +156,19 @@ complete(myEntity).ok().done();
 The following is an example
 
 ```
-var sdk = require('kinvey-backend-sdk');
-var service = sdk.service(function(err, service) {
-  var dataLink = service.dataLink;   // gets the datalink object from the service
-  var notImplementedHandler = function(request, complete) {
+let sdk = require('kinvey-backend-sdk');
+let service = sdk.service((err, service) => {
```

Comment:
Here and the rest of the documentation,let's use standard functions rather than arrow functions, since many people may/may not be familiar with them yet.